src/assets/style/module/blackBox/_blackBox.less

```diff
@@ -0,0 +1,40 @@
+/*
+    These are designed for use with "black-boxed" content - that is,
+    we generate the HTML from something else (i.e. markdown), so we
+    shouldn't modify it.
+
+    Because the HTML is generated from markdown, we can't assign classes
+    to it directly, so we have to use element selectors.
+*/
+.blackBox h1,
```

Comment:
IMHO the term `.blackBox` is too ambiguous. I would prefer something like `.showdown` to make it more obvious where the markup is coming from and to distinguish it from other generated content.

Reply:
I agree. Already pressed the button :grimacing: we can get it some other time I guess.